packages/p2p/src/pages/buy-sell/buy-sell-table.jsx

```diff
@@ -48,6 +49,12 @@ const BuySellTable = ({ onScroll }) => {
     const { error, has_more_items_to_load, isError, isLoading, loadMoreAdverts, rendered_adverts } =
         useP2PRenderedAdverts();
 
+    React.useEffect(() => {
+        if (error?.code === api_error_codes.PERMISSION_DENIED) {
+            general_store.setIsBlocked(true);
+        }
+    }, [error?.code]);
+
```

Comment:
previously we checked p2p_advertiser_info response for Permission Denied error, but now we don't call it if the user is not a p2p user, so need this check